convention-develocity-gradle-plugin/plugins/gradle-5-or-newer/src/main/java/com/myorg/ConventionDevelocityGradlePlugin.java

```diff
@@ -83,9 +95,27 @@ private void configureBuildCache(BuildCacheConfiguration buildCache, DevelocityC
         });
     }
 
-    private static boolean isCi() {
+    private boolean isCi() {
         // CHANGE ME: Apply your environment detection logic here
-        return System.getenv().containsKey("CI");
+        return environmentVariable("CI").isPresent();
```

Comment:
Do containsKey("CI") and isPresent() have the same semantics, particularly when the key is present but the value is null or the empty string?

Reply:
Yes. If defined but with an empty string, `System.getenv("CI")` will return an empty string. An empty string is still considered valued and therefore "present" in the eyes of `Optional`.

`System.getenv("CI")` will only return `null` if the `CI` environment variable is not defined in the system. 

This reflects the semantics of `containsKey`.

Reply:
Great. Thanks for the confirmation.